membership/views.py

```diff
@@ -0,0 +1,20 @@
+from django.shortcuts import render
+
+
+from django.views.generic import ListView
+from membership.models import Membership, UserMembership, Subscription
+# Create your views here.
+
+class MembershipView(ListView):
+    model = Membership
+    template_name = 'memberships/list.html'
+    def get_user_membership(self):
+        user_membership_qs = UserMembership.objects.filter(user=self.request.user)
+        if user_membership_qs.exists():
+            return user_membership_qs.first()
+        return None
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(**kwargs)
+        current_membership = self.get_user_membership(self.request)
```

Comment:
You're passing in `self.request` into a method on a class with only one argument. 

Python methods (functions on classes) _always_ take `self` as its first parameter, but you never pass anything into it. It's automatically assumed `self` is passed in. 

Instead you would write: 

```python
def get_user_membership(self, request):
    user_membership_qs = UserMembership.objects.filter(user=request.user)
    ...
```